src/applications/personalization/profile-2/tests/components/connected-apps/ConnectedAppDeleteModal.unit.spec.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+
+import { ConnectedAppDeleteModal } from '../../../components/connected-apps/ConnectedAppDeleteModal';
+
+describe('<ConnectedAppDeleteModal>', () => {
+  it('renders correctly when not deleting', () => {
+    const defaultProps = {
+      title: 'hello',
+      deleting: false,
+      modalOpen: true,
+      onCloseModal: () => {},
+      onConfirmDelete: () => {},
+    };
+
+    const wrapper = mount(<ConnectedAppDeleteModal {...defaultProps} />);
+
+    const text = wrapper.text();
+    expect(text).to.include('Do you want to disconnect this app?');
+    expect(text).to.include(
+      'hello won’t have access to new information about you from VA once you disconnect. This may impact the usefulness of the app.',
+    );
+    expect(text).to.include('Cancel');
+    expect(text).to.include('Disconnect');
```

Comment:
these two assertions could be checking to make sure there were buttons with these labels rather than just making sure the text appeared anywhere in the rendered component.